packages/@lightningjs/ui-components/src/components/Card/CardRadio.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright 2023 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import CardTitle from './CardTitle';
+import TextBox from '../TextBox';
+import * as styles from './CardRadio.styles.js';
+import Radio from '../Radio';
+
+export default class CardRadio extends CardTitle {
+  static get __componentName() {
+    return 'CardRadio';
+  }
+
+  static get __themeStyle() {
+    return styles;
+  }
+
+  static get properties() {
+    return [...super.properties, 'radio', 'subtitle'];
+  }
+
+  static get tags() {
+    return [...super.tags, 'Radio', 'Subtitle'];
+  }
+
+  static _template() {
+    return {
+      ...super._template(),
+      Subtitle: {
+        type: TextBox,
+        signals: {
+          textBoxChanged: '_update'
+        }
+      }
+    };
+  }
+
+  _update() {
+    super._update();
+    this._updateSubtitle();
+    this._updateRadio();
+    this._updateDescriptionPosition();
+    this._updateSubtitlePosition();
+  }
+
+  _updateSubtitle() {
+    this._Subtitle.patch({
+      content: this.subtitle,
+      style: {
+        textStyle: {
+          ...this.style.subtitleTextStyle,
+          wordWrapWidth: this._calculateTextWidth()
+        }
+      }
+    });
+  }
+
+  _updateRadio() {
+    const { iconWidth, iconHeight } = this.style;
+    const radioPatch = {
+      ...this.radio,
+      mode: this.mode,
+      w: iconWidth,
+      h: iconHeight,
```

Comment:
`icon` is a misleading term here. Unless the designers specified that the radio should be a larger size on the CardRadio component than the normal Radio component, then I don't think this is necessary anyway since the user can pass in something like this
```js
CardRadio: {
  radio: {
    style: { w: 40, h: 40 }
  }
}
```

If the height DOES need to be increased per design specs, then let's just add a nested radio object in styles so we would have something like this in this updateRadio method:
```js
  const radioPatch = {
    ...this.styles.radio,
    ...this.radio,
    mountX: 1
    x: this.w - this.style.paddingHorizontal,
    y: this.style.paddingVertical
  };
```
(that Y position is also overly complicated, it should just be the padding value)

Reply:
And honestly, if the size does need to change for Xfinity design specs, we should just do that in the Xfinity theme and I would remove that altogether here, so you'd only have:
```js
  const radioPatch = {
    ...this.radio,
    mountX: 1
    x: this.w - this.style.paddingHorizontal,
    y: this.style.paddingVertical
  };
```